src/main/java/org/fundacion/jala/converter/core/facade/CompilerFacade.java

```diff
@@ -37,7 +34,7 @@ public CompilerFacade() {
      * @param code is string with code in java.
      * @return a String of the result on runtime console.
      */
-    public static String facadeJavaCompile(final String code) {
+    public static String facadeJavaCompile(final String code) throws CompilerException {
         if (!code.isBlank() || !code.equals(null)) {
             javaCompiler = new JavaCompiler();
             String filePath = Transform.toFile(code, "Main", "java");
```

Comment:
"Main" and "Java" could be final attributes.

Reply:
Sorry, that's code I haven't done